book/starting-installing.md

```diff
@@ -86,15 +86,54 @@ We also recommend that you install _git_, as this will be needed to get many of
 smart install git
 ```
 
-### Installing packages into a _virtualenv_
+### Installing packages using the Solo client
 
-This section shows how to install and run packages in an isolated Python environment using  [virtualenv](https://virtualenv.pypa.io/en/latest/).
+The recommended way of working with Python and DroneKit-Python is to use the
+[Solo CLI](starting-utils.html#deployingrunning-dronekit-scripts-on-solo) 
+to package and run the script in a virtual environment.
 
 <aside class="note">
-Solo uses many globally installed packages that may be out of date (in particular, [DroneKit](example-dronekit.html)). Using _virtualenv_ means that you don't need to update the global versions; potentially disturbing native Solo code.
+Solo uses many globally installed packages that may be out of date (in particular, [DroneKit](example-dronekit.html)). Using an isolated environment means that you don't need to update the global versions; potentially disturbing native Solo code.
 </aside>
 
-First install _virtualenv_ on Solo using _pip_:
+The CLI takes care of packaging all the scripts in a folder:
```

Comment:
suggest:
 +The CLI takes care of packaging all the scripts in your current folder: